diff --git a/browser/components/tabbrowser/content/tabgroup.js b/browser/components/tabbrowser/content/tabgroup.js
index 670fe2b309863102f63246e26194c8958e28bbb7..4cf64a3f4b04542ae8c792cf526b823e8c967953 100644
--- a/browser/components/tabbrowser/content/tabgroup.js
+++ b/browser/components/tabbrowser/content/tabgroup.js
@@ -12,7 +12,9 @@
       <vbox class="tab-group-label-container" pack="center">
         <label class="tab-group-label" role="button"/>
       </vbox>
-      <html:slot/>
+      <vbox class="tab-group-container">
+        <html:slot/>
+      </vbox>
       `;
 
     /** @type {string} */
@@ -213,7 +215,7 @@
     }
 
     get tabs() {
-      return Array.from(this.children).filter(node => node.matches("tab"));
+      return Array.from(this.querySelector('.tab-group-container').children).filter(node => node.matches("tab"));
     }
 
     /**
